karma.conf.js

```diff
@@ -96,5 +93,12 @@ module.exports = function (config) {
         // Concurrency level
         // how many browser should be started simultaneous
         concurrency: Infinity,
+
+        customLaunchers: {
+            ChromeHeadlessCustom: {
```

Comment:
What does this do? No idea lol

Reply:
This is setting it up so we can use Chrome as a headless browser instead of PhantomJS.

It is great because we don't even need to bring any extra dependency, the same chrome launcher supports headless and regular mode!